### PR TITLE
Documentation: update cli comments

### PR DESCRIPTION
**Pull request by Gilthoniel**

### CODE REVIEW

File: cli/node/example_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package node
```
Comment (nkcr) [inline cli/node/example_test.go:1]: 👌

**Review by nkcr — APPROVED** [review]

File: cli/node/example_test.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+// daemon side.
+type Hello interface {
+	SayTo(name string)
+}
+
+type simpleHello struct{}
+
+func (simpleHello) SayTo(name string) {
+	fmt.Printf("Hello, %s!", name)
+}
```
Comment (pierluca) [inline cli/node/example_test.go:48]: this logic is not demonstrated in the example though and is just dead (example) code, or am I missing something?
I'm a tiny bit confused

**Review by pierluca — COMMENTED** [review]
Tiny remark on the example, but overalls 👍 👍 👍

File: cli/node/example_test.go (hunk 2 above)
Comment (Gilthoniel) [reply cli/node/example_test.go:48]: To demonstrate, I need to start a daemon and that's quite complex in the example as it should be an independent command but I wanted to show how to write the controller.

**Review by pierluca — APPROVED** [review]
LGTM

File: cli/node/example_test.go (hunk 2 above)
Comment (pierluca) [reply cli/node/example_test.go:48]: Fair enough :)

**Pull request merged**